src/serviceWorker.js

```diff
@@ -11,125 +11,125 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
-    window.location.hostname === 'localhost' ||
+  window.location.hostname === 'localhost'
```

Comment:
this looks wrong

Reply:
I think it breaks it up because of line length and the comments

Reply:
I think the `||` got removed though because the next line is a comment